Doc/library/io.rst

```diff
@@ -915,11 +915,19 @@ Text I/O
 
       .. versionadded:: 3.7
 
-   .. method:: reconfigure(*, line_buffering=None, write_through=None)
+   .. method:: reconfigure(*, encoding, errors, newline, \
```

Comment:
The signature should make it clear that parameters are optional (e.g. `encoding=None` or whatever the default value is).

Reply:
Default value is just a placeholder to detect "not specified".
It's different between Pure Python implementation and C implementation.
How can I document it?

Reply:
Is `None` supported in both cases? If yes, use `None`.

Reply:
Not passing ``newline`` means "keep current newline".
On the other hand, ``newline=None`` means "reset to default newline".

Reply:
Then perhaps `reconfigure(*[, encoding][, errors][, newline][, line_buffering][, write_through])`?